stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/app.py

```diff
@@ -43,6 +44,11 @@
 app = api.app
 
 
+@app.on_event("startup")
+async def _startup_event():
+    IndexesClient().create_indexes()
```

Comment:
Forgive my ignorance, but what state is the app in while creating indexes? Is it unusable? Or just slow? Curious if startup should block while the app is starting.

Reply:
it hasn't started yet.  In ES, an index is the same as a database table in an RDBMS, rather than like creating an index in an RDBMS.

It will take a couple seconds to run this the very first time it starts, and then all subsequent starts are milliseconds since it's a no-op.